GOVERNANCE.md

```diff
@@ -124,11 +127,30 @@ the issue tracker is:
 
 ## Collaborator nominations
 
-Existing collaborators can nominate someone to become a collaborator. Nominees
-should have significant and valuable contributions across the Node.js
+### Who can nominate Collaborators?
+
+Existing Collaborators can nominate someone to become a Collaborator.
+
+### Ideal Nominees
+
+Nominees should have significant and valuable contributions across the Node.js
 organization.
 
-To nominate a new collaborator, open an issue in the [nodejs/node][] repository.
+Contributions can be:
+
+* Commits
+* Pull requests
+* Comments
+* Reviews
```

Comment:
I think it's a bit confusing to list both Commits and PRs, same for Reviews and PRs
```suggestion
* Opening pull requests.
* Comments and reviews.
```